lib/java/persistance-dsl/src/main/java/org/enso/persist/impl/PersistableProcessor.java

```diff
@@ -66,24 +69,50 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
     if (roundEnv.processingOver()) {
       for (var entry : registeredClasses.entrySet()) {
         try {
+          var props = new Properties();
+          var propsWhere = StandardLocation.SOURCE_OUTPUT;
+          var propsPkg = entry.getKey();
+          var propsName = "Persistables.properties";
```

Comment:
d9b49110b0 improves the incremental compilation behavior by:
- storing previously known registrations into `Persistables.properties` file
- this file is only generated into `src_managed` directory - e.g. not part of the final JAR
- when incremental compilation triggers compilation of **not whole package**
- we still read all the previous `Persistables.properties` registrations and merge them in
- as a result the generated `Persistables.java` accumulates all the known results
- after `clean` the properties file is removed and we start from scratch ...
- ... by compiling whole package and seeing all the registrations by the annotation processor